HISTORY.rst

```diff
@@ -1,6 +1,29 @@
 =======
 History
 =======
+3.19.0 (2020-09-17)
```

Comment:
I see we've missed this the past few releases, but the release number should have a line under it to turn it into a heading on https://cumulusci.readthedocs.io/en/latest/history.html

Like this: https://github.com/SFDO-Tooling/CumulusCI/pull/2030/files#diff-d9ae695c8119e58c2ec435b6deae2d53R107-R108